internal/olm/client/client.go

```diff
@@ -280,18 +310,16 @@ func (c Client) printPodErrors(ctx context.Context, depSelectors *metav1.LabelSe
 		return fmt.Errorf("error getting Pods: %v", err)
 	}
 	for _, p := range podList.Items {
-		if p.Status.Phase != corev1.PodSucceeded {
-			for _, cs := range p.Status.ContainerStatuses {
-				if !cs.Ready {
-					podErrors[p.Name] = cs.State.Waiting.Message
-				}
```

Comment:
Should we not check if the Pod is NOT Running before here?

Reply:
In the earlier design, we were checking. However, from what I saw, the pod can be failing, even when PHASE= running. Hence, that check did not make any sense.